mk/setup.mk

```diff
@@ -22,6 +22,7 @@ clean:
 	rm -rf mypyreport
 	rm -rf src/prodsec
 	rm -rf venv
```

Comment:
Not a big deal but I wonder whether there is any `venv` ever expected to be created by the standard project setup. If there was some random other `venv` I would rather not remove it during the cleanup - the author is responsible for cleaning any non-standard stuff.